src/F0_output.py

```diff
@@ -74,14 +112,18 @@ def evaluate_dict(dict_values):
         plots.capacities(
             dict_values["simulation_settings"],
             dict_values["project_data"],
-            dict_values["kpi"]["cost_matrix"]["label"],
+            dict_values["kpi"]["scalar_matrix"]["label"],
             dict_values["kpi"]["scalar_matrix"]["optimizedAddCap"],
         )
+    return
```

Comment:
```suggestion
```

If you return `None` you don't need to add `return`

Reply:
Yeah... I know. I have this everywhere, don´t I? If I change it I need to change it all, so for now I will leave it as it is.